scripts/apps/contacts/components/ItemContainer.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {find} from 'lodash';
+
+export function ItemContainer(props) {
+    var item = props.item;
+    var key = props.field || null;
+    var label, value, title, altTitle;
+    var _class = 'container';
+    var _link = _class + ' link';
+
+    const {gettextCatalog} = props.svc;
+
+    function getPhoneValue(item) {
```

Comment:
React components can be classes, no need to define functions inline here. Make a class that extends `React.Component` and define classes as methods.